changelog/635.feature.rst

```diff
@@ -0,0 +1,27 @@
+Implement new cloned attributes on the :func:`clone` method of:
+
+    - :class:`TextChannel`:
+        The :meth:`TextChannel.clone` will now clone :attr:`TextChannel.topic`, :attr:`TextChannel.position`, :attr:`TextChannel.nsfw`,
+        :attr:`TextChannel.category`, :attr:`TextChannel.slowmode_delay`, :attr:`TextChannel.type`, :attr:`TextChannel.default_auto_archive_duration`,
+        :attr:`TextChannel.flags` into the newly-cloned TextChannel. In addition, :meth:`TextChannel.clone` now accepts the listed attributes as kwargs
+        to make an in-place edit of the newly-cloned channel.
```

Comment:
```suggestion
    - :meth:`TextChannel.clone` will now clone :attr:`~TextChannel.topic`, :attr:`~TextChannel.position`, :attr:`~TextChannel.nsfw`,
      :attr:`~TextChannel.category`, :attr:`~TextChannel.slowmode_delay`, :attr:`~TextChannel.type`, :attr:`~TextChannel.default_auto_archive_duration`,
      :attr:`~TextChannel.flags` into the newly-cloned channel.
```
Something like this, perhaps? The last sentence (`... now accepts the listed attributes as kwargs ...`) in all of these could be mentioned just once at the top, no need to add it to each paragraph.